dev/export-datameer-env.sh

```diff
@@ -0,0 +1,7 @@
+export __vHadoop='hadoop-2.7'
+export __vScala='scala-2.12'
+export __vDatameer='datameer-11.1-gcs'
+export __distSuffix=${__vHadoop}-${__vScala}-${__vDatameer}
+export __dockerTag='2.4.2'-${__distSuffix}
+export __dockerRegistry=us.gcr.io/devops-project-200915
```

Comment:
if the names given by spark env?
It looks a bit unusual for environment variables

Reply:
no just used us. i choosed the double underscore to make sure noone else is using the same vars.